src/app/pages/home/home.module.ts

```diff
@@ -7,13 +7,6 @@ import { Routes, RouterModule } from '@angular/router';
 import { HomePageRoutingModule } from './home-routing.module';
 import { HomePage } from './home.page';
 
-const routes: Routes = [
-  {
-    path:'',
-    component:HomePage
```

Comment:
Not sure how this works with the home routing module in place. I'd prefer to keep routing modules in side module files for simplicity but it works fine just like this.